test/EntityFramework6.Npgsql.Tests/Support/EntityFrameworkTestBase.cs

```diff
@@ -213,6 +223,85 @@ private static DbCompiledModel CreateModel(NpgsqlConnection connection)
                 null);
             dbModel.StoreModel.AddItem(echoFunc);
 
+            var modelBlogStoreType = dbModel.StoreModel.EntityTypes.First(x => x.Name == typeof(Blog).Name) as EntityType;
+            var rowType = RowType.Create(
+                modelBlogStoreType.Properties.Select(x =>
+                {
+                    var clone = EdmProperty.Create(x.Name, x.TypeUsage);
+                    clone.CollectionKind = x.CollectionKind;
+                    clone.ConcurrencyMode = x.ConcurrencyMode;
+                    clone.IsFixedLength = x.IsFixedLength;
+                    clone.IsMaxLength = x.IsMaxLength;
+                    clone.IsUnicode = x.IsUnicode;
+                    clone.MaxLength = x.MaxLength;
+                    clone.Precision = x.Precision;
+                    clone.Scale = x.Scale;
+                    clone.StoreGeneratedPattern = x.StoreGeneratedPattern;
+                    clone.SetMetadataProperties(x
+                        .MetadataProperties
+                        .Where(metadataProerty => !clone
+                            .MetadataProperties
+                            .Any(cloneMetadataProperty => cloneMetadataProperty.Name.Equals(metadataProerty.Name))));
+                    return clone;
+                }),
+                null);
+
+            var getBlogsFunc = EdmFunction.Create(
+                "StoredGetBlogsFunction",
+                "BloggingContext",
+                DataSpace.SSpace,
+                new EdmFunctionPayload
+                {
+                    ParameterTypeSemantics = ParameterTypeSemantics.AllowImplicitConversion,
+                    Schema = "dbo",
+                    IsComposable = true,
```

Comment:
Shouldn't this be IsComposable = false ? Or you can use TVF inside LINQ ? (I'm not familiar with TVFs in general, so I'm just making sure).